### PR TITLE
Cpa 522 - Default landing Page Option

### PR DESCRIPTION
**Pull request by dylanj1752**
## 🗣 Description

Added default landing page option to landing page dropdown on template selection. Will use the default landing page at time of campaign creation when this is selected. 

If no landing page is selected (left blank) it will automatically use the default landing page. A landing page is no longer a required field on the template editor

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/api/utils/aws_utils.py [hunk 1]
```diff
@@ -30,7 +30,24 @@ def upload_fileobj_image(self, data):
         key = f"{uuid.uuid4().hex}.png"
         logger.info(f"data={data} bucket={self.image_bucket} key={key}")
         self.client.upload_fileobj(data, self.image_bucket, key)
+<<<<<<< HEAD
```
Comment (zenine07) [inline src/api/utils/aws_utils.py:33]: Is that supposed to be here?

File: src/api/utils/aws_utils.py [hunk 2]
```diff
@@ -30,7 +30,24 @@ def upload_fileobj_image(self, data):
...
+        # if self.endpoint_url:
+        #     parsed_url = urlparse(self.endpoint_url)
+        #     external_host = os.environ.get("AWS_S3_EXTERNAL_HOST", "localhost")
+        #     host = f"{parsed_url.scheme}://{external_host}:{parsed_url.port}"
+        # else:
+        #     host = "https://s3.amazonaws.com"
+
+=======
         host = "https://s3.amazonaws.com"
+>>>>>>> develop
```
Comment (zenine07) [inline src/api/utils/aws_utils.py:50]: Same with this. looks like some weird merge results

**Review by zenine07 — CHANGES_REQUESTED** [review]
Just the comments on the lines I mentioned. Looks like a bad merge or something.

**Comment by dylanj1752** [comment]
Oddities with merge request. All necessary changes will be included in next push, closing this one

**Pull request closed without merging**